### DISCUSSION TITLE
Add support for killing queries

### DISCUSSION THREAD

**Discussion by wooque** [planetscale/discussion#448] [Feature Requests]
`KILL QUERY 123456` now throws `syntax error at position 5 near 'kill'`

**Comment by orware** [comment]
Hi @wooque,

You are correct on this topic and it is on the issue tracker for Vitess currently here:
https://github.com/vitessio/vitess/issues/5497

**Reply by aviraccoon** [reply]
This was added in https://github.com/vitessio/vitess/pull/13371, first appeared in v0.18.0-rc1. Should the [MySQL compatibility page](https://planetscale.com/docs/reference/mysql-compatibility#queries-functions-syntax-data-types-and-sql-modes) be updated?
